src/lib/sql/sql_translator.cpp

```diff
@@ -134,15 +134,17 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::translate_statement(const hsql::
 
 std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_insert(const hsql::InsertStatement& insert) {
   const std::string table_name{insert.tableName};
+
+  AssertInput(StorageManager::get().has_table(table_name), std::string("Insert: Invalid table name: " + table_name));
+
   auto target_table = StorageManager::get().get_table(table_name);
 
-  Assert(target_table != nullptr, "Insert: Invalid table name");
 
   std::shared_ptr<AbstractLQPNode> current_result_node;
 
   // Check for SELECT ... INTO .. query
   if (insert.type == hsql::kInsertSelect) {
-    DebugAssert(insert.select != nullptr, "Insert: no select statement given");
+    AssertInput(insert.select != nullptr, std::string("Insert: no select statement given"));
```

Comment:
Why do we have to create an `std::string` here?

Reply:
Otherwise, it will be handled as const char* and we can't add the "Invalid input error: " extension to the output message in src/lib/utils/assert.hpp . Or do you think we don't need to add the extension every time and it should be  part of the initial string?

Reply:
AssertInput should be able to take a string literal, pls do all necessary conversions/use std::stringstream in AssertInput.